client/src/components/EditPlace.js

```diff
@@ -71,7 +127,8 @@ export class EditPlace extends React.Component {
               <InputTextarea
                 rows={5}
                 cols={30}
-                value={fakePlace.description}
+                value={this.state.description}
+                onChange={e => this.setState({ description: e.target.value })}
```

Comment:
Don't use arrow function in render (this `displayForm` is used in render). In general just avoid using arrow function in JSX if you are concerned about performance

Reply:
Sorry, I know what you taught me a few days ago, I didn't deliberately ignore that haha. I just changed it to arrow functions because we didn't like the impure push history functions. I'll change it.

So just making sure, I'll make separate functions to handle all the click events and push history?